app/views/alaveteli_pro/info_request_batches/_authority_list.text.erb

```diff
@@ -0,0 +1,11 @@
+<% if public_bodies.size %>
+<% example_body_names = public_bodies.take(2).map(&:name) %>
+<%= n_('{{count}} recipient, {{first_public_body}}',
```

Comment:
I've fixed this issue up in c2824bb, 9188ab3, e57c020 and 071e57a. In the end, I duplicated the code from this view back into the html version, with appropriate calls to `raw` or `html_safe` in the text version. I also went through all the text email templates and made sure (with lots of new specs) that they were escaping everything they should.